### ISSUE TITLE
Is there any option to add a new driver without modifying and recompiling the project?

### ISSUE THREAD

**Issue by jozsavcsaba** [rncbc/qjackctl#183]

**Comment by rncbc** [comment]
nope; what noew driver you're talking about? it must be one that is "built-in" to JACK, otherwise I don't know what you mean

**Comment by jozsavcsaba** [comment]
Yeah, the one made by me! :)

**Comment by jozsavcsaba** [comment]
I think it isn't a big deal...

**Comment by rncbc** [comment]
if you enable the JACK D-BUS interface and start qjackctl that way, the driver list is dynamic as reported by jack(ctl API) itself; the formerly hard-coded driver names (that better called JACK "back-ends") are not listed whatsoever

**Comment by jozsavcsaba** [comment]
A script option would be also preferable. I don't have time now, but maybe i could do it...

**Comment by jozsavcsaba** [comment]
Sorry i didn't see that you wrote. So then i have to enable jackdbus. And sorry again, im very new with all these things, i do not understand any component, and how are they working together, but i'm on it, i will learn!

**Comment by jozsavcsaba** [comment]
I mean i do not understand all components, sorry!

**Comment by jozsavcsaba** [comment]
Ok, so what i tryed is i copied the driver file into the drivers installation folder then i restarted jackdbus. If i list the drivers by jackd -h or jack_control dl i see my driver and i can use it from command line with jackdbus and jackd also, but the list in the qjackctl does not change... I think i missunderstood something.

**Comment by jozsavcsaba** [comment]
So i think it could be made with like an option to add a new driver file, and all the changable driver parameters to appear dinamically in a list.
